### DISCUSSION TITLE
Keyboard shortcuts for Quarto in VSCode not working "command not found"

### DISCUSSION THREAD

**Discussion by lyonsdm** [quarto-dev/quarto-cli#7278] [Q&A]
### Description

I have the Quarto extension installed in VSCode. None of the default keyboard shortcuts for Quarto work. For example pressing ctrl+enter will not run my current line, and vscode gives message "Command 'quarto.runCurrent'" not found. The keybindings are present in my json file. Similar keyboard shortcuts all work for .rmd files, but not .qmd. I would like to use Quarto. Does anyone know how to fix this?

I have tried re-installing Quarto extension.

**Comment by mcanouil** [comment]
You should not need to have defined the keyboard shortcuts as it is part of the default shortcut from the extension.

<img width="1110" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/81e7604c-99db-47a0-980e-76a9a3e2feac">

You could try using the command palette.
<img width="606" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/66da6ef2-e415-486b-8bd5-bc30e901b7ef">


Note that without the means to reproduce your case, it's going to be hard to help as in several setup including Codespaces, it just works out of the box, meaning it's specific to your setup which we don't know anything about at this time.
